ajax/libs/bootstrap-combobox/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bootstrap-combobox",
+  "filename": "js/bootstrap-combobox.min.js",
+  "description": "A combobox plugin that works with twitter bootstrap",
+  "version": "1.1.7",
+  "author": {
+    "name": "Daniel Farrell",
+    "url": "https://github.com/danielfarrell"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/danielfarrell/bootstrap-combobox"
+  },
+  "keywords": [
+    "bootstrap-combobox",
+    "bootstrap",
+    "combobox"
+  ],
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/danielfarrell/bootstrap-combobox.git",
+    "basePath": "",
+    "files": [
+      "+(js|css)!(tests)/bootstrap-combobox*"
```

Comment:
@PeterDaveHello I tried this config locally and I can grab the files exclude ```tests``` folder under ```js``` folder.
Is there another good way to do this?